examples/gno.land/r/demo/mail/state.gno

```diff
@@ -0,0 +1,19 @@
+package mail
+
+import (
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/releases"
+	"std"
+)
+
+// realm state
+var (
+	addr2Mailbox             = avl.Tree{}                                 // Address -> *Mailbox
+	counter      int         = 0                                          // number of mails ever sent
+	admin        std.Address = "g1fjh9y7ausp27dqsdq0qrcsnmgvwm6829v2au7d" // @grepsuzette
+
+	Fee std.Coin = std.Coin{
+		Denom:  "ugnot",
+		Amount: 250 * 1000, // should never cost more $0.12 or 0.25
+	}
+)
```

Comment:
I don't think we should be separating out state from the base file where that state is actively being used - it just creates confusion on where it's coming from. What do you think about moving this snippet closer to where it's being used?

Reply:
I certainly lack experience here, I see Thomas agrees with you.
I'll let you know my thought to have a chance to understand why you disagree. For me:

* State is the most central thing for a realm. 
* The code gravitates around it and is interchangeable. 
* Therefore knowing what is state is key to approaching a realm.

Approaching one realm, I always look for the part where the state is defined. 
The most obvious label for me for the state is to look for a file called `state.gno`.
This way it's possible to open state.gno in one split and work on something else.
If a folder doesn't contain one, I will respect the author choice, but then I will have to open a lot of files until I find it. But in fact, I don't even argue to have files called a certain way, I argue authors should have their own style; I think diversity is good for the project, like in evolution.